feat: add wecom qrcode authentication

wenchao-h opened the pull request:
- 增加企业微信扫码登录模板文件和样式
- 同时支持企业微信扫码和蓝鲸账号密码登录

vincohuang committed:
feat: add wecom qrcode authentication

- 增加企业微信扫码登录模板文件和样式
- 支持企业微信扫码和蓝鲸账号密码登录

vincohuang committed:
feat: add wecom qrcode authentication

- 增加企业微信扫码登录模板文件和样式
- 支持企业微信扫码和蓝鲸账号密码登录

tencent-adm commented:
[![CLA assistant check](https://cla.opensource.tencent.com/pull/badge/not_signed)](https://cla.opensource.tencent.com/Tencent/bk-PaaS?pullRequest=496) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-PaaS?pullRequest=496) before we can accept your contribution.<br/><hr/>**vincohuang** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-PaaS?pullRequest=496) it.</sub>

wenchao-h committed:
feat: minify css and js file

wenchao-h committed:
update: remove wecom settings from default.py

    - add wecom settings in settings_development.py
    - add wecom comment in settings_login.py

wenchao-h committed:
update: wecom login view code refactoring

wenchao-h committed:
Merge branch 'dev-wecom' into develop

    - remove wecom settings from default.py
    - minify css and js files
    - wecom login view code refactoring
